docs/sources/operators-guide/configure/configuring-out-of-order-samples-ingestion.md

```diff
@@ -9,19 +9,7 @@ weight: 120
 
 # Configuring out-of-order samples ingestion
 
-Grafana Mimir and the Prometheus TSDB understand out-of-order as follows.
-
-The moment that a new series sample arrives, Mimir need to determine if the series already exists, and whether or not the sample is too old:
-
-- If the series exists, the incoming sample must have a newer timestamp than the latest sample that is stored for the series.
-  Otherwise, it is considered out-of-order and will be dropped by the ingesters.
-- If the series does not exist, then the sample has to be within bounds, which go back 1 hour from TSDB's head-block max time (when using 2 hour block range). If it fails to be within bounds, then it is also considered out-of-bounds and will be dropped by the ingesters.
-
-> **Note:** If you're writing metrics using Prometheus remote write or the Grafana Agent, then out-of-order samples are unexpected.
-> Prometheus and Grafana Agent guarantee that samples are written in-order for the same series.
-
-If you have out-of-order samples due to the nature of your architecture or the system that is being observed, then you can configure Grafana Mimir to set an out-of-order time-window threshold for how old samples can be ingested.
-As a result, none of the preceding samples will be dropped if they are within the configured time window.
+Grafana Mimir allows ingesting out-of-order samples from 2.2 release onwards. It is currently an experimental feature.
```

Comment:
```suggestion
As an experimental feature, Mimir allows you to ingest out-of-order samples.
```
* Remove reference to version because the docs themselves are versioned.
* Remove embedded future promises.